spec/lib/github_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe Github do
+  it "creates oauth client" do
+    expect(Octokit::Client).to receive(:new).with(client_id: Config.github_app_client_id, client_secret: Config.github_app_client_secret)
+
+    described_class.oauth_client
+  end
+
+  it "creates app client" do
+    expect(Config).to receive(:github_app_id).and_return("123456")
+    expect(Config).to receive(:github_app_private_key).and_return(<<PRIVATE_KEY)
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
We use it as `OpenSSL::PKey::RSA.new(Config.github_app_private_key)`, so it validates it. 

I took this key from another test case.

Reply:
Can we mock that part to avoid using real looking keys?

`expect(OpenSSL::PKey::RSA).to receive(:new)`

Normally, I would recommend changing the other tests (in a separate PR), but it seems other tests specifically tests if we can format/render ssh keys, so it fits the purpose.

Reply:
I mocked it